docs/docs/reference/release-notes/v5.13/index.md

```diff
@@ -0,0 +1,80 @@
+---
+date: "2023-12-@TODO"
+version: "5.13.0"
+title: "v5.13 Release Notes"
+---
+
+Welcome to `gatsby@5.13.0` release (December 2023 #1)
+
+Key highlight of this release:
+
+- [Custom Image and File CDN URL generators](#custom-image-and-file-cdn-url-generators)
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v5.12)
```

Comment:
links starting with `/docs` will not work in github - release notes are meant to be rendered on gatsbyjs.com site where those relative links would work (https://www.gatsbyjs.com/docs/reference/release-notes/v5.12/ is example of release notes being rendered there and "previous release notes" on that also use link like that https://github.com/gatsbyjs/gatsby/blob/eed07f3da59cd3d94b9e8c0b12aa1570f8a99a60/docs/docs/reference/release-notes/v5.12/index.md#L17 and in rendered version it works correctly)